Bindings/bindings.json

```diff
@@ -12,21 +12,22 @@
             "displayName": "Timer",
             "direction": "trigger",
             "defaultParameterName": "myTimer",
+            "parameterNamePrompt":"Name the timestamp object for use in your code",
             "settings": [
                 {
                     "name": "schedule",
                     "value": "string",
                     "defaultValue": "0 * * * * *",
                     "required": true,
-                    "label": "How often would you like to run? Please provide a cron expression.",
-                    "help": "A cron express is a way of expressing a schedule. See the docs to learn more."
+                    "label": "How frequently should this function run? Enter a cron expression.",
+                    "help": "A cron expression is a way of expressing a schedule. Please consult the documentation to learn more."
                 },
                 {
                     "name": "runOnStartup",
                     "value": "boolean",
                     "defaultValue": false,
-                    "label": "Run on start up?",
-                    "help": "When true, your function will run on start. Useful for debugging."
+                    "label": "Should this function run on start up?",
+                    "help": "When true, your function will run on startup. This may be useful for debugging."
```

Comment:
Hmmm, we should think more about the notion of "startup". While it makes sense in core SDK continuous/always on scenarios, I don't think "startup" is part of the conceptual model for Functions. Also not sure about the note about debugging.